build.sbt

```diff
@@ -97,7 +98,7 @@ lazy val sharedSettings = Seq(
 lazy val `odin-core` = (project in file("core"))
   .settings(sharedSettings)
   .settings(
-    libraryDependencies ++= (monix % Test) :: catsMtl :: sourcecode :: monixCatnap :: perfolation :: catsEffect :: cats
+    libraryDependencies ++= (catsEffect % Test) :: catsMtl :: sourcecode :: perfolation :: catsEffectStd :: cats
```

Comment:
You don't need to add `catsEffectStd` if you have core already.

Reply:
@kubukoz I want to have `IO` only in tests, since `cats-effect-std` is enough for the core.